discord/bot.js

```diff
@@ -90,7 +90,13 @@ const destroyTasks = () => {
 const commands = [
     {
         name: "shop",
-        description: "Show your current daily shop!"
+        description: "Show your current daily shop!",
+        options: [{
+            type: "USER",
+            name: "user",
+            description: "From wich user you will see the daily shop?",
```

Comment:
This sentence is somehow strange. Either cross out the question mark or reformulate it.